packages/gafl-webapp-service/src/processors/__tests__/date-and-time-display.spec.js

```diff
@@ -13,28 +13,36 @@ describe('displayStartTime', () => {
     const startTime = displayStartTime({ licenceStartDate: '2021-01-01', licenceStartTime: '12' })
     expect(startTime).toEqual('Friday, January 1st, 2021, 12:00pm (midday)')
   })
-  it('displays the date in the correct format where the start time is 3pm', () => {
+  it('displays the date in the correct format where the start time is 3pm (UTC)', () => {
     const startTime = displayStartTime({ licenceStartDate: '2021-01-01', licenceStartTime: '15' })
     expect(startTime).toEqual('Friday, January 1st, 2021, 3:00pm')
   })
+  it('displays the date in the correct format where the start time is 3pm (BST)', () => {
+    const startTime = displayStartTime({ licenceStartDate: '2021-07-01', licenceStartTime: '15' })
+    expect(startTime).toEqual('Thursday, July 1st, 2021, 3:00pm')
+  })
```

Comment:
Not sure what is differentiating this BST test from the UTC test above?

Reply:
It's because the date used falls into summertime (whereas the similar test above used Jan 1st as the date)

Reply:
https://www.youtube.com/watch?v=uuN7kOW7iwo